src/rest-server/package.json

```diff
@@ -60,6 +61,15 @@
     "winston": "~2.4.0",
     "xml2js": "~0.4.19"
   },
+  "_moduleAliases": {
+    "@config": "src/config",
```

Comment:
Why not create an alias of root directory `src/`?

Reply:
P.S. webpack also has a built-in module alias feature https://webpack.js.org/configuration/resolve/#resolvealias

Reply:
it's OK to create an alias for `src` but the path will be longer?

we don't use `webpack` in rest server

Reply:
Have you tried `NODE_PATH` environment variable?

Reply:
> Have you tried `NODE_PATH` environment variable?

seems it does not work on windows using dotenv or command inline env